### PR TITLE
@DeviaVir request: Added logic for post only orders (handled differently by Kraken), added Kraken to README and removed ; at end of lines

### PR DESCRIPTION
**Pull request by crubb**
As per @DeviaVir in https://github.com/carlos8f/zenbot/pull/207 ;)

### CODE REVIEW

File: extensions/kraken/update-products.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+var KrakenClient = require('kraken-api');
```
Comment (DeviaVir) [inline extensions/kraken/update-products.sh]: forgot the `;`'s in this file though ;)

**Comment by crubb** [comment]
Grr ;)

**Review by DeviaVir — APPROVED** [review]
:+1: sweet!

**Comment by crubb** [comment]
Oh no, I'm sorry, I seem to have sent the PR to the `master` branch, can you merge it into `feat/kraken`? Otherwise it won't swap over to https://github.com/carlos8f/zenbot/pull/207. Do I need to send a new one?

**Comment by Alexandre-io** [comment]
Damn, never use your smartphone to accept a merge request :/ Could you create a new one ? sorry ..

**Pull request merged**